frontend/src/queries/nodes/DataTable/exportTransformers.ts

```diff
@@ -0,0 +1,47 @@
+import { removeExpressionComment } from '~/queries/nodes/DataTable/utils'
+import { DataTableNode, EventsQuery } from '~/queries/schema/schema-general'
+import { isEventsQuery } from '~/queries/utils'
+
+const PERSON_COLUMN = 'person'
+const PERSON_EMAIL_COLUMN = 'person.properties.email'
+
+/**
+ * Replaces the person column with the email column for performance reasons
+ */
+export function transformColumnsForExport(columns: string[]): string[] {
+    return columns.map((column) => {
+        const cleanColumn = removeExpressionComment(column)
+
+        // Replace 'person' with 'person.properties.email' for performance
```

Comment:
not all persons have email, which columns were before and its not exported anymore?

Reply:
all columns are already replaced by `person.properties.email` by default. This was done a while ago.

However, the `query.source` is still not replacing that column, which causes the export to fail